modules/cnf-image-based-upgrade-seed-image-config.adoc

```diff
@@ -87,6 +87,9 @@ The following table lists the components, resources, and configurations that you
 |All Day 2 Operator subscriptions
 |Yes
 
+|`ClusterLogging.yaml`
```

Comment:
`ClusterLogging.yaml` was removed with CLO 6.0 so I think we should not show it here either,  see https://github.com/openshift-kni/cnf-features-deploy/pull/2045 as reference.